packages/core/src/config.js

```diff
@@ -372,12 +372,13 @@ export const comparisonSchema = {
         osVersion: { type: 'string' },
         width: {
           type: 'integer',
-          maximum: 2000,
-          minimum: 120
+          minimum: 1,
```

Comment:
min width 1px is allowed?

Reply:
Yes, we have `> 0` validation on API. This is because the App percy does not depend on what is the screensize of the device, we just collect this metadata to show in UI so that users can see it. In case SDK fails to collect this info, a default 1px, 1px width, height is used